oshi-core/src/main/java/oshi/software/os/unix/aix/AixFileSystem.java

```diff
@@ -169,4 +170,18 @@ public long getOpenFileDescriptors() {
     public long getMaxFileDescriptors() {
         return ParseUtil.parseLongOrDefault(ExecutingCommand.getFirstAnswer("ulimit -n"), 0L);
     }
+
+    @Override
+    public long getMaxFileDescriptorsPerProcess() {
+        final List<String> lines = FileUtil.readFile("/etc/security/limits");
+        for (final String line : lines) {
+            if (line.startsWith("nofiles(descriptors)")) {
+                final String[] split = line.split("\\w+");
```

Comment:
I'm not sure if this is the best delimiter choice, and compiling regexes other than a single character is less performant.  There are some pre-compiled regexes in `ParseUtil` like `ParseUtil.whitespaces` or `ParseUtil.notDigits` that you can use, or this may be a good candidate for `ParseUtil.getLastInt()`.

Do you need to `trim()` the string before splitting?  [This example](https://sites.ualberta.ca/dept/chemeng/AIX-43/share/man/info/C/a_doc_lib/files/aixfiles/limits.htm) (from AIX 4) shows indented values per-user, although I haven't found an example file for AIX 7 which we should use the format for.

Reply:
`ParseUtil.getLastLong()` seems to be a good fit. 👍🏻 

I think you are correct. [This 7.2 reference](https://www.ibm.com/docs/en/ssw_aix_72/filesreference/filesreference_pdf.pdf) looks exactly as the AIX 4 example I will add a `trim()` and `startsWith("nofiles")` check.

Reply:
You'll also need to check if it matches the current user name...